### PR TITLE
[intel-ipsec-mb]Add intel-ipsec-mb package

### PR DESCRIPTION
**Pull request by kreuzerkrieg**
Intel(R) Multi-Buffer Crypto for IPsec Library

### CODE REVIEW

**Comment by kreuzerkrieg** [comment]
I know it wouldnt compile, it requires NASM 2.13.03 or newer. How do we solve such problems?

File: ports/intel-ipsec/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,52 @@
+INCLUDE(vcpkg_common_functions)
+
+INCLUDE(vcpkg_common_functions)
```
Comment (Rastaban) [inline ports/intel-ipsec/portfile.cmake]: included twice

**Comment by Rastaban** [comment]
```
~$ nasm --version
NASM version 2.13.02
```
As you predicted I see nasm related failures.  I'm okay with accepting a port that fails on our CI machines for now. At some point ubuntu will pick up an updated version and it will get installed and start passing.  In the mean time individuals like you will be able to take advantage of the port.  The down side is that for now this port won't have test coverage and if ports are added that have a dependacy on this one they will be skipped in the build tests.

Can you add a message to the portfile that points out there is a dependancy on nasm 2.13.03 or newer?  Bonus points if you can figure out how to get it to fail the build with a clear error message if an older version is installed.

**Comment by kreuzerkrieg** [comment]
> ```
> ~$ nasm --version
> NASM version 2.13.02
> ```
> As you predicted I see nasm related failures. I'm okay with accepting a port that fails on our CI machines for now. At some point ubuntu will pick up an updated version and it will get installed and start passing. In the mean time individuals like you will be able to take advantage of the port. The down side is that for now this port won't have test coverage and if ports are added that have a dependacy on this one they will be skipped in the build tests.
> 
> Can you add a message to the portfile that points out there is a dependancy on nasm 2.13.03 or newer? Bonus points if you can figure out how to get it to fail the build with a clear error message if an older version is installed.

I like your "bonus points" stuff :) will try to do it.
so, there is no way to bring newer version and install it locally? I mean, NASM has all needed binaries, if there is a mechanism to download, unpack and somehow use with in your port without affecting builds of another ports it will be perfect
As of Ubuntu catching up with NASM, when they will backport NASM 2.14 to Ubuntu 18.04 this package will need NASM 3.0 :)

**Comment by kreuzerkrieg** [comment]
fixed and pushed

**Pull request merged**